README.md

```diff
@@ -140,18 +140,25 @@ import { authMiddleware } from '@descope/nextjs-sdk/server'
 export default authMiddleware({
 	// The Descope project ID to use for authentication
 	// Defaults to process.env.DESCOPE_PROJECT_ID
-	projectId: 'your-descope-project-id'
+	projectId: 'your-descope-project-id',
 
 	// The URL to redirect to if the user is not authenticated
 	// Defaults to process.env.SIGN_IN_ROUTE or '/sign-in' if not provided
 	// NOTE: In case it contains query parameters that exist in the original URL, they will override the original query parameters. e.g. if the original URL is /page?param1=1&param2=2 and the redirect URL is /sign-in?param1=3, the final redirect URL will be /sign-in?param1=3&param2=2
-	redirectUrl?: string
+	redirectUrl?: string,
 
 	// An array of public routes that do not require authentication
+	// All routes are private by default, use this to configure routes that don't require authentication
 	// In addition to the default public routes:
 	// - process.env.SIGN_IN_ROUTE or /sign-in if not provided
 	// - process.env.SIGN_UP_ROUTE or /sign-up if not provided
-	publicRoutes?: string[]
+	publicRoutes?: string[],
+
+	// An array of private routes that require authentication
+	// If defined, the specified route will require authentication and the rest of the routes be will be public
+	privateRoutes?: string[],
+
+	// If you having privateRoutes and publicRoutes defined at the same time, privateRoutes will be ignored.
```

Comment:
not critical, but once we have more properties here it will look weird 
I would comment this together with privateRoutes comment

Reply:
I just added to the README afterwards. The comments were getting too messy in the `options` code.